packages/product-components/src/components/SelectAssetModal/NetworkTabs.tsx

```diff
@@ -0,0 +1,138 @@
+import { AssetLogo, Row, useElevation } from '@trezor/components';
+import {
+    borders,
+    Elevation,
+    mapElevationToBackground,
+    mapElevationToBorder,
+    spacings,
+    spacingsPx,
+    typography,
+} from '@trezor/theme';
+import { SelectAssetNetworkProps, SelectAssetSearchCategoryType } from './SelectAssetModal';
+import styled, { DefaultTheme } from 'styled-components';
+import { FormattedMessage } from 'react-intl';
+
+interface CheckableTagProps {
+    $variant: 'primary' | 'tertiary';
+    $elevation: Elevation;
+}
+
+const getCheckableTagStyles = (
+    $elevation: Elevation,
+    theme: DefaultTheme,
+    type: CheckableTagProps['$variant'] | 'hover',
+) => {
+    switch (type) {
+        case 'primary':
+            return `
+                background: ${theme.backgroundPrimarySubtleOnElevation1};
+                color: ${theme.textPrimaryDefault};
+                border: 1px solid ${theme.borderSecondary};
+        `;
+        case 'hover':
+            return `
+                background: ${mapElevationToBackground({ $elevation, theme })};
+                color: ${theme.textDefault};
+                border: 1px solid ${mapElevationToBorder({ $elevation, theme })};
+            `;
+        default:
+            return `
+                background: ${mapElevationToBackground({ $elevation, theme })};
+                color: ${theme.textSubdued};
+                border: 1px solid ${mapElevationToBackground({ $elevation, theme })};
+            `;
+    }
+};
+
+// temporary styles, should be replaced in the future
+const CheckableTag = styled.button<CheckableTagProps>`
```

Comment:
Could you please extract this component and related logic to different file? 🙏  we will later refactor it to the design system

Reply:
It is done